### ISSUE TITLE
[Feature] Support Helm deployment

### ISSUE THREAD

**Issue by chipzoller** [kubecost/disk-autoscaler#9]
### Problem Statement

Users who need customization would like to use Helm to deploy DAS.

### Solution Description

Support deployment via a Helm chart.

### Troubleshooting

- [x] I have searched other issues in this repository and mine is not recorded.

**Comment by dwbrown2** [comment]
I think it would be really cool to see this easily deployed and configured with the Kubecost helm. But maybe there's a standalone operator too? Thoughts?

**Comment by chipzoller** [comment]
I don't think we should inflate the cost-analyzer chart more than it already is. This is a side project and it's totally fine to install a second chart for ancillary projects like this.
